docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -138,6 +138,9 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+The methods IndicesQueryBuilder#parseNoMatchQuery(String) and IndicesQueryBuilder#noMatchQuery(String) have been removed,
```

Comment:
If I remember correctly we had discussed this, and decided to remove it in favor of using IndicesQueryBuilder#noMatchQuery(QueryBuilder).